examples/example_utils.cpp

```diff
@@ -17,73 +17,74 @@ mip::Timestamp getCurrentTimestamp()
-#else  // MIP_USE_EXTRAS
+        example_utils->connection = std::unique_ptr<RecordingTcpConnection>(
+            new RecordingTcpConnection(example_utils->recordedFile.get(), example_utils->recordedFile.get(), port_or_hostname, port));
+#else // !MIP_USE_EXTRAS
         using TcpConnection = mip::platform::TcpConnection;
         example_utils->connection = std::unique_ptr<TcpConnection>(new TcpConnection(port_or_hostname, port));
-#endif  // MIP_USE_EXTRAS
+#endif // MIP_USE_EXTRAS
 
-        example_utils->device = std::unique_ptr<mip::DeviceInterface>(new mip::DeviceInterface(example_utils->connection.get(), example_utils->buffer, sizeof(example_utils->buffer), 1000, 2000));
-#else  // MIP_USE_TCP
+        example_utils->device = std::unique_ptr<mip::DeviceInterface>(
+            new mip::DeviceInterface(example_utils->connection.get(), example_utils->buffer, sizeof(example_utils->buffer), 1000,
+                2000));
+#else // !MIP_USE_TCP
         throw std::runtime_error("This program was compiled without socket support. Recompile with -DMIP_USE_TCP=1");
 #endif // MIP_USE_TCP
-
     }
     else  // Serial port
     {
-
 #ifdef MIP_USE_SERIAL
         uint32_t baud = std::strtoul(baud_or_port.c_str(), nullptr, 10);
-        if( baud == 0 )
-        {
+        if (baud == 0)
             throw std::runtime_error("Serial baud rate must be a decimal integer greater than 0.");
-        }
 
 #ifdef MIP_USE_EXTRAS
         using RecordingSerialConnection = mip::extras::RecordingConnectionWrapper<mip::platform::SerialConnection>;
-        example_utils->connection = std::unique_ptr<RecordingSerialConnection>(new RecordingSerialConnection(example_utils->recordedFile.get(), example_utils->recordedFile.get(), port_or_hostname, baud));
-#else  // MIP_USE_EXTRAS
+        example_utils->connection = std::unique_ptr<RecordingSerialConnection>(
+            new RecordingSerialConnection(example_utils->recordedFile.get(), example_utils->recordedFile.get(), port_or_hostname,
+                baud));
+#else // !MIP_USE_EXTRAS
         using SerialConnection = mip::platform::SerialConnection;
         example_utils->connection = std::unique_ptr<SerialConnection>(new SerialConnection(port_or_hostname, baud));
-#endif  // MIP_USE_EXTRAS
-        example_utils->device = std::unique_ptr<mip::DeviceInterface>(new mip::DeviceInterface(example_utils->connection.get(), example_utils->buffer, sizeof(example_utils->buffer), mip::C::mip_timeout_from_baudrate(baud), 500));
-#else  // MIP_USE_SERIAL
+#endif // MIP_USE_EXTRAS
+        example_utils->device = std::unique_ptr<mip::DeviceInterface>(
+            new mip::DeviceInterface(example_utils->connection.get(), example_utils->buffer, sizeof(example_utils->buffer),
+                mip::C::mip_timeout_from_baudrate(baud), 500));
+#else // !MIP_USE_SERIAL
```

Comment:
Extra indent on the last line (`mip_timeout_from_baudrate`). The closing `);` should go on the next line, with the same indent as the opening line. Like with { } braces. In this case I think it would also be cleaner to put all 3 args on their own line. This is how I'd write it:
```
        example_utils->device = std::unique_ptr<mip::DeviceInterface>(
            new mip::DeviceInterface(example_utils->connection.get(),
            example_utils->buffer, sizeof(example_utils->buffer),
            mip::C::mip_timeout_from_baudrate(baud), 500)
        );
```